### ISSUE TITLE
Segmentation with 3x8-bit pixels Images in Cityscapes

### ISSUE THREAD

**Issue by bpinaya** [dusty-nv/jetson-inference#249]
Hi there @dusty-nv , I was trying to get the segmentation example to work with a different dataset. For labeling, I used the same tools as CityScapes, but changed the colors of my labels.
The problem is that sometimes when training on Digits I get the `Check failed: status == CUBLAS STATUS_SUCCESS (11 vs. 0) CUBLAS STATUS MAPPING_ERROR` error, somehow expanded [here](https://github.com/alexgkendall/caffe-segnet/issues/101), it seemed to be the PIL conversion (to generate the labels I use the same script with the difference of changing RGBA to RGB) and [the modes](https://pillow.readthedocs.io/en/3.1.x/handbook/concepts.html#concept-modes).
The error itself was very tricky to find, I think it has to do with the colors I'm using and the values of the generated labels. So my question is:
- To train Cityscapes on Digits, how did you generate the labels? Using the same script? Any other changes?

**Comment by bpinaya** [comment]
Note that if I use the 'P' mode in PIL to generate the labes, I got the :`ERROR: Exception: ValueError: All label images must use the same palette`
So I'm  wondering how did you proceed to the training in Digits of Cityscapes.

**Comment by bpinaya** [comment]
Also, as weird as it may sound it works with different portions of my dataset, I think some color might be the cause of this, but I find that super tricky. I tried with a subset of my data (about 15 images/labels) and works good, tried it with a bigger subset (about 30-50) and the problem appears. In my opinion, it might be the label generation.
In the meanwhile, I switched to pytorch but I really wanted to get it to work on caffe and digits, because of the visualizations and so.
